services/QuillLMS/client/app/bundles/Teacher/components/dashboard/welcome_modal.tsx

```diff
@@ -1,10 +1,59 @@
 import * as React from 'react'
 import Confetti from 'react-confetti'
 
-const classroomActivitiesSrc = `${process.env.CDN_URL}/images/pages/dashboard/illustrations-classroom-activities.svg`
+import { handleHasAppSetting } from "../../../Shared/utils/appSettingAPIs";
+
+const classroomActivitiesSrc = `${process.env.CDN_URL}/images/pages/dashboard/illustrations-classroom-activities-2.svg`
+
+const exploreDemoLink = `${process.env.DEFAULT_URL}/teachers/view_demo`
+
+const WelcomeModalWithoutDemo = ({close}) => (
+  <div className="welcome-modal-without-demo quill-modal">
+    <div className="modal-body">
+      <h1>Welcome to Quill!</h1>
+      <div className="welcome-modal-text">
+        <p>Our mission as a non-profit is to help students become strong writers, so all our content is completely free to use with an unlimited number of students.</p>
+      </div>
+      <button className="quill-button contained primary medium" onClick={close} type="button">Let&apos;s go!</button>
+    </div>
+    <img alt="Teacher at projector in classroom" src={classroomActivitiesSrc} />
+  </div>
+)
+
+const WelcomeModalWithDemo = ({close}) => (
+  <div className="welcome-modal-with-demo quill-modal">
+    <div className="modal-head">
+      <button className="close-welcome-modal" onClick={close} type="button"><img alt="Close the modal" src={`${process.env.CDN_URL}/images/shared/close_x.svg`} /></button>
+    </div>
+
+    <div className="modal-body">
+
+      <div className="welcome-modal-image-box">
+        <h1>Welcome to Quill!</h1>
+        <img alt="Teacher at projector in classroom" src={classroomActivitiesSrc} />
+      </div>
+      <div>
+        <div className="welcome-modal-text">
+          <p className="welcome-modal-header">Want to see Quill&apos;s full potential?</p>
+          <p>Play around with a demo account to see sample student data and reports.</p>
+        </div>
+        <br />
+        <div className="welcome-modal-buttons">
+          <a className="quill-button contained secondary medium outlined welcome-modal-demo" href={exploreDemoLink}>Explore demo</a>
+          <button className="quill-button contained primary medium welcome-modal-skip" onClick={close} type="button">Maybe later, skip</button>
```

Comment:
I think that since we want to encourage people to click this, maybe we should swap these buttons so that the "Explore demo" button is the "primary contained" class and comes after the "Maybe later, skip" button? Not sure if this is something that got mocked up previously or not. Unrelated: it would be nice to put `focus-on-light` classes on both of these buttons for accessibility purposes.

Reply:
This was a discussion I had with Tom C before he left (because he made this mockup). I think his reasoning was that we still want to emphasize the option that lets teachers get started as quickly as possible (which is "Skip"). We only want to encourage the users who are actually interested in exploring the demo to click it, instead of having everyone automatically explore the demo without realizing they're going into a demo experience, if that makes sense?

I will put the `focus-on-light` class on both buttons!